### ISSUE TITLE
'lag' when sending group messages

### ISSUE THREAD

**Issue by iurimatias** [status-im/nim-status-client#1367]
in large group chats the UI gets stuck momentarily when sending a message, this is likely because for each user in the private group chat a different message is generated in parallel and this is done sync. even so, we fix this so at least the UI doesn't freeze when sending the message

**Comment by iurimatias** [comment]
duplicate of #1315
